config/crd/bases/mutations.gatekeeper.sh_assign.yaml

```diff
@@ -106,6 +106,8 @@ spec:
                         description: matchLabels is a map of {key,value} pairs. A single {key,value} in the matchLabels map is equivalent to an element of matchExpressions, whose key field is "key", the operator is "In", and the values array contains only "value". The requirements are ANDed.
                         type: object
                     type: object
+                  name:
+                    type: string
```

Comment:
Add `description`s for the `name` field in this and other CRDs\.

Reply:
Done